build/ci/ansible/roles/kubefate/files/clean.sh

```diff
@@ -1,18 +1,35 @@
 #/bin/bash
 
-clean_when_timeout()
+clean()
 {
   rm -rf ${BASE_DIR}/*
 
-  # Install Kind
+  # Delete kind
   kind_status=`kind version`
   if [ $? -eq 0 ]; then
     echo "Deleting kind cluster..." 
     kind delete cluster
   fi
 
-  echo "exit because of task timeout"
-  exit 1
+  # delete docker containers
+  docker stop $(docker ps -a -q)
+  docker rm $(docker ps -a -q)
+
+  # delete docker images
+  docker rmi -f $(docker images -q)
```

Comment:
I think is it fine to only remove images with `${DOCKER_REGISTRY}federatedai/` prefix

Reply:
done